.codesandbox/tasks.json

```diff
@@ -1,4 +1,17 @@
 {
+  // These tasks will run in order when initializing your CodeSandbox project.
```

Comment:
Comments in `json`, does this not throw any errors? 👀

Reply:
pitcher removes comments before deserializing the config file. these have been added a while ago to help users navigate the json file better

Reply:
~Shouldn't it be a `.jsonc` file then?~ [Doesn't look like it needs to be an explicit `.jsonc` file](https://code.visualstudio.com/docs/languages/json#_json-with-comments). So I guess we're good.